docs/4-At-The-Internship/getting_most_out_of_internship.md

```diff
@@ -0,0 +1,19 @@
+## Getting the most out of your internship
+
+New experience, new opportunities!
+
+### Outside of work
+
+- If you're in a new city, exploring can be really fun!
+- Meetup.com to meet new people, share interests and learn new things!
+- A nice thing about jobs v.s. school is that you don't have homework when you get back home. This means it's a great time to pick up a new hobby, especially recurring ones
+
+> e.g. (Rudi) I started rock climbing during an internship, the mix of exercise, problem solving, and non-competitiveness makes this sport appealing to a lot of techies
+> I also like to look on Groupon for events and things to do in my area. While the main purpose of the site is discounts, it does a pretty good job at aggregating that sort of stuff.
+
+### At work
+
+Since you might be working on larger codebases than you're used to or you might get to work in a team with other people, having a couple extra learning goals can help you get the most out of your internship.
+
+- Ask to [pair program](https://en.wikipedia.org/wiki/Pair_programming) with more experienced engineers once in a while, to learn how they think, what tricks they have and what tools of the trade they use
```

Comment:
what do you think about linking to https://content.pivotal.io/blog/pair-programming-considered-extremely-beneficial instead?